yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -43,11 +43,6 @@ class FlowableOperator : public Flowable<D> {
         : flowable_(std::move(flowable)), subscriber_(std::move(subscriber)) {
       assert(flowable_);
       assert(subscriber_);
-
-      // We expect to be heap-allocated; until this subscription finishes (is
-      // canceled; completes; error's out), hold a reference so we are not
-      // deallocated (by the subscriber).
-      Refcounted::incRef(*this);
```

Comment:
Can you add that fix to this PR too? The line that we were passing `*this` to a constructor, using the Subscriber&. Should we pass yarpl::Reference<Subscriber> in that Emitter class?

I could not be able to see the benefit of manually calling inc_ref, dec_ref other than delayed destruction, for which we already have a comment stating that the original implementor wants to have it. I am not sure if that comment still has validity. Can you sync up with @lehecka ?